cfme/utils/appliance/__init__.py

```diff
@@ -2040,7 +2041,7 @@ def server_roles(self, roles):
         self.update_advanced_settings({'server': server_data})
         timeout = 600 if enabling_ansible else 300
         wait_for(lambda: self.server_roles == roles, num_sec=timeout, delay=15)
-        if enabling_ansible:
+        if enabling_ansible and self.version < '5.11':
```

Comment:
I don't think these version checks are necessary. You can call `wait_for_embedded_ansible` on either appliance version, and the changes to `is_embedded_ansible_running` will handle the version changes.

Reply:
I will update `wait_for_embedded_ansible` change and remove other unnecessary changes.